### PR TITLE
allow players to play blue in ladder

### PR DESCRIPTION
**Pull request by smallst**
image

here to battle in right team

### CODE REVIEW

File: app/lib/LevelLoader.coffee [hunk 1]
```diff
@@ -204,8 +207,18 @@ module.exports = class LevelLoader extends CocoClass
         console.debug 'LevelLoader: loading opponent session:', @opponentSession if LOG
         @listenToOnce @opponentSession, 'sync', @preloadTokenForOpponentSession
 
+  prelaodTeamForSession: (session) =>
```
Comment (nwinter) [inline app/lib/LevelLoader.coffee:210]:
```suggestion
  preloadTeamForSession: (session) =>
```

File: app/lib/LevelLoader.coffee [hunk 2]
```diff
@@ -190,12 +193,12 @@ module.exports = class LevelLoader extends CocoClass
...
       console.debug 'LevelLoader: session already loaded:', @session if LOG
       @session.setURL '/db/level.session/' + @session.id
-      @loadDependenciesForSession @session
+      @prelaodTeamForSession @session
     else
       console.debug 'LevelLoader: loading session:', @session if LOG
       @listenToOnce @session, 'sync', ->
         @session.setURL '/db/level.session/' + @session.id
-        @loadDependenciesForSession @session
+        @prelaodTeamForSession @session
```
Comment (nwinter) [inline app/lib/LevelLoader.coffee:201]:
```suggestion
        @preloadTeamForSession @session
```

File: app/lib/LevelLoader.coffee [hunk 3]
```diff
@@ -190,12 +193,12 @@ module.exports = class LevelLoader extends CocoClass
     if @session.loaded
       console.debug 'LevelLoader: session already loaded:', @session if LOG
       @session.setURL '/db/level.session/' + @session.id
-      @loadDependenciesForSession @session
+      @prelaodTeamForSession @session
```
Comment (nwinter) [inline app/lib/LevelLoader.coffee:196]:
```suggestion
      @preloadTeamForSession @session
```

**Comment by smallst** [comment]
good point, find a bug when spectate.

**Pull request merged**